src/components/radio-button/radio-button/radio-button.scss

```diff
@@ -121,10 +121,20 @@
   }
 }
 
+slot[name='suffix'] {
```

Comment:
Is this really needed? normally slot has display: contents so it's not really there, and if there is no content, there is really nothing there.